bobtemplates/plone_addon/src/+package.namespace+/+package.name+/profiles/default/types/+package.dexterity_type_name+.xml.bob

```diff
@@ -1,44 +1,82 @@
-<object name="{{{ package.dexterity_type_name }}}" meta_type="Dexterity FTI" i18n:domain="{{{ package.dottedname }}}"
-   xmlns:i18n="http://xml.zope.org/namespaces/i18n">
-  <property name="title" i18n:translate="">{{{ package.dexterity_type_name }}}</property>
-  <property name="description"
-    i18n:translate=""></property>
-  <property name="icon_expr"></property>
+<object
+    i18n:domain="{{{ package.dottedname }}}"
+    meta_type="Dexterity FTI"
+    name="{{{ package.dexterity_type_name }}}"
+    xmlns:i18n="http://xml.zope.org/namespaces/i18n">
+  <!-- Basic properties -->
+  <property
+      i18n:translate=""
+      name="title">{{{ package.dexterity_type_name }}}</property>
+  <property
+      i18n:translate=""
+      name="description"></property>
   <property name="factory">{{{ package.dexterity_type_name }}}</property>
   <property name="add_view_expr">string:${folder_url}/++add++{{{ package.dexterity_type_name }}}</property>
+  <property name="icon_expr"></property>
   <property name="link_target"></property>
-  <property name="immediate_view">view</property>
+  <property name="allow_discussion">False</property>
+  <!-- Hierarchy control -->
   <property name="global_allow">True</property>
   <property name="filter_content_types">True</property>
   <property name="allowed_content_types">
+    <!-- <element value="" />-->
   </property>
-  <property name="allow_discussion">False</property>
-  <property name="default_view">view</property>
-  <property name="view_methods">
-    <element value="view"/>
-  </property>
-  <property name="default_view_fallback">False</property>
-  <property name="add_permission">cmf.AddPortalContent</property>
+  <!-- Schema, class and security -->
   <property name="klass">plone.dexterity.content.Item</property>
-  <property name="behaviors">
-    <element value="plone.app.content.interfaces.INameFromTitle"/>
-  </property>
+  <property name="add_permission">cmf.AddPortalContent</property>
   <property name="schema">{{{ package.dottedname }}}.interfaces.I{{{ package.dexterity_type_name }}}</property>
   <property name="model_source"></property>
   <property name="model_file"></property>
-  <alias from="(Default)" to="(dynamic view)"/>
-  <alias from="edit" to="@@edit"/>
-  <alias from="sharing" to="@@sharing"/>
-  <alias from="view" to="(selected layout)"/>
-  <action title="View" action_id="view" category="object" condition_expr=""
-    description="" icon_expr="" link_target="" url_expr="string:${object_url}"
-    visible="True">
-    <permission value="View"/>
+  <!-- Enabled behaviors -->
+  <property name="behaviors">
+    <element value="plone.app.content.interfaces.INameFromTitle" />
+  </property>
+  <!-- View and aliases -->
+  <property name="immediate_view">view</property>
+  <property name="default_view">view</property>
+  <property name="view_methods">
+    <element value="view" />
+  </property>
+  <property name="default_view_fallback">False</property>
```

Comment:
Good point. While looking at this, do we really need a alias for sharing and edit - or is this historic?